lib/src/network/http.dart

```diff
@@ -135,14 +183,30 @@ String makeUserAgent(PackageInfo info, BaseDeviceInfo deviceInfo, String sri, Li
 
 /// A [Client] that logs all requests.
 class LoggingClient extends BaseClient {
```

Comment:
I think we can now make it private. And rename it since it's no more a `LoggingClient`. Something like `_RegisterCallbackClient`?